TemplateApplication/TemplateApplicationStandard.swift

```diff
@@ -125,4 +127,21 @@ actor TemplateApplicationStandard: Standard, ObservableObject, ObservableObjectP
             logger.error("Could not delete user document: \(error)")
         }
     }
+    
+    /// Stores the given consent form in the user's document directory with a unique timestamped filename.
+    ///
+    /// - Parameter consent: The consent form's data to be stored as a `PDFDocument`.
+    func store(consent: PDFDocument) async {
+        guard let basePath = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first else {
+            logger.error("Could not create path for writing consent form to user document directory.")
+            return
+        }
+        
+        let formatter = DateFormatter()
+        formatter.dateFormat = "yyyy-MM-dd_HHmmss"
+        let dateString = formatter.string(from: Date())
+        
+        let filePath = basePath.appending(path: "consentForm_\(dateString).pdf")
+        consent.write(to: filePath)
```

Comment:
I think this is a great behavior if we have disable the Firebase interaction. I think the Firebase behaviour should be uploading the PDF to firebase in e.g. the firebase cloud storage: https://firebase.google.com/docs/storage